soc/arc/snps_nsim/Kconfig.defconfig.hs6x

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Synopsys, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NSIM_HS6X
+
+config CPU_HS6X
+	default y
+
+config NUM_IRQ_PRIO_LEVELS
+	# This processor supports 16 priority levels:
+	# 0 for Fast Interrupts (FIRQs) and 1-15 for Regular Interrupts (IRQs).
```

Comment:
We don't currently support FIRQ in ARCv3 Zephyr port, so let's get rid of this comment.

Reply:
Done, thanks!